### PR TITLE
Fix: Show an indication when enumerating folder with lots of files

### PR DESCRIPTION
**Pull request by rbadapanda**

### CODE REVIEW

**Comment by rbadapanda** [comment]
Thanks stevenguh@ for suggesting a quick way to solve this issue.  
This CR implements his suggestion.

**Comment by rbadapanda** [comment]
> Thanks stevenguh@ for suggesting a quick way to solve the issue @ https://github.com/bodil/vscode-file-browser/issues/20 

This CR implements his suggestion.

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "publisher": "bodil",
     "displayName": "File Browser",
     "description": "A nicer alternative to the file open dialog.",
-    "version": "0.2.10",
+    "version": "0.2.11",
```
Comment (bodil) [inline package.json]: Please don't bump versions in PRs, that should be done by the release process.

**Comment by bodil** [comment]
Thanks; merging. 👍

**Comment by stevenguh** [comment]
Just notice this, this unreleased change has a bug where the indicator will appear in the action menu. Also somehow the `package.json` was moved.

**Pull request merged**